astropy/utils/iers/iers.py

```diff
@@ -40,13 +40,13 @@
 
 # IERS-A default file name, URL, and ReadMe with content description
 IERS_A_FILE = 'finals2000A.all'
-IERS_A_URL = 'https://maia.usno.navy.mil/ser7/finals2000A.all'
-IERS_A_URL_MIRROR = 'https://datacenter.iers.org/data/9/finals2000A.all'
+IERS_A_URL = 'https://datacenter.iers.org/data/9/finals2000A.all'
+IERS_A_URL_MIRROR = 'https://maia.usno.navy.mil/ser7/finals2000A.all'
 IERS_A_README = get_pkg_data_filename('data/ReadMe.finals2000A')
 
 # IERS-B default file name, URL, and ReadMe with content description
-IERS_B_FILE = get_pkg_data_filename('data/eopc04_IAU2000.62-now')
-IERS_B_URL = 'http://hpiers.obspm.fr/iers/eop/eopc04/eopc04_IAU2000.62-now'
+IERS_B_FILE = 'eopc04_14_IAU2000.62-now'
```

Comment:
I think `IERS_B_FILE` points to a local file in https://github.com/astropy/astropy/tree/main/astropy/utils/iers/data . Do you really have to change this one?

Reply:
yeah, the different filenames between the mirrors is annoying. is there a reason `IERS_B` points to packaged data (which may be well out of date) while `IERS_A` does not and is always downloaded?

Reply:
anyway, looks like a deeper can of worms there so i'll revert it and leave `IERS_B` alone for now.